src/routes/signin/providers/utils.ts

```diff
@@ -103,8 +107,9 @@ const manageProviderStrategy =
     }
 
     if (email) {
-      // check email
-      if (!(await isValidEmail({ email }))) {
+      try {
+        await emailValidator.validateAsync(email);
+      } catch {
         return done(new Error('email is not allowed'));
```

Comment:
Ideally, this error should result in a redirect to the client's app to show a nice error message. I'm not sure it's possible given we don't have access to `res` here.

Reply:
Yes, this is something we should improve, I haven't found a simple way to implement this behaviour